src/compas_fab/robots/robot.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, model, artist=None, semantics=None, client=None):
         self._scale_factor = 1.
         self.model = model
         self.attached_tool = None
-        self.artist = artist  # setter and getter (because of scale)
+        self._artist = artist  # setter and getter (because of scale)
```

Comment:
No need to set `self.artist` until it's referenced.

Reply:
This doesn't seem right; the `artist` setter does a few things related to scaling and what not, if you skip it, when does that happen?